src/ServiceInsight/ServiceControl/DefaultServiceControl.cs

```diff
@@ -537,6 +564,7 @@ void LogError(IRestResponse response)
         static bool HasSucceeded(IRestResponse response) => successCodes.Any(x => response != null && x == response.StatusCode && response.ErrorException == null);
 
         static IEnumerable<HttpStatusCode> successCodes = new[] { HttpStatusCode.OK, HttpStatusCode.Accepted, HttpStatusCode.NotModified, HttpStatusCode.NoContent };
+        static Regex linkExpression = new Regex(@"<([^>]+)>;\s?rel=""(\w+)"",?\s?", RegexOptions.Compiled);
```

Comment:
Unfortunately not because `,` is used as a delimiting character  but can also be present in the URL, e.g. `http://localhost:33333/api/messages?offsets=0,1,2,3; rel="next", http://localhost:33333/....`.